lib/prx_access.rb

```diff
@@ -1,22 +1,29 @@
 module PRXAccess
```

Comment:
perhapst his could be the start of a gem to make these common methods available, and make it easier to call prx apis via ruby.